[2025-02-05T12:17:15.502+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T12:17:15.512+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.localiza_dw_test manual__2025-02-05T12:16:26+00:00 [queued]>
[2025-02-05T12:17:15.517+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.localiza_dw_test manual__2025-02-05T12:16:26+00:00 [queued]>
[2025-02-05T12:17:15.518+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T12:17:15.526+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): model_sales.localiza_dw_test> on 2025-02-05 12:16:26+00:00
[2025-02-05T12:17:15.529+0000] {standard_task_runner.py:63} INFO - Started process 2840 to run task
[2025-02-05T12:17:15.531+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_sales', 'model_sales.localiza_dw_test', 'manual__2025-02-05T12:16:26+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_sales.py', '--cfg-path', '/tmp/tmpw0jmxm7f']
[2025-02-05T12:17:15.533+0000] {standard_task_runner.py:91} INFO - Job 85: Subtask model_sales.localiza_dw_test
[2025-02-05T12:17:15.566+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_sales.model_sales.localiza_dw_test manual__2025-02-05T12:16:26+00:00 [running]> on host 98025f4a542d
[2025-02-05T12:17:15.625+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_sales' AIRFLOW_CTX_TASK_ID='model_sales.localiza_dw_test' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T12:16:26+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T12:16:26+00:00'
[2025-02-05T12:17:15.626+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T12:17:17.247+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T12:17:17.248+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpa5lik8_a from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T12:17:17.258+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_sales__model_sales/target/partial_parse.msgpack
[2025-02-05T12:17:17.262+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T12:17:17.268+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T12:17:17.271+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf/profiles.yml.
[2025-02-05T12:17:17.272+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['test', '--select', 'path:seeds path:models/marts/sales', '--vars', '{"PROD_1": P004,                                 "PROD_2": P002,                                 "DATA_INICIO": 1999-01-01,                                 "DATA_FIM": 2025-02-05}', '--project-dir', '/tmp/tmpa5lik8_a', '--profiles-dir', '/tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf', '--profile', '***', '--target', 'prod']
 in /tmp/tmpa5lik8_a
[2025-02-05T12:17:17.303+0000] {logging_mixin.py:188} INFO - 12:17:17  Running with dbt=1.6.1
[2025-02-05T12:17:17.399+0000] {logging_mixin.py:188} INFO - 12:17:17  Registered adapter: postgres=1.6.1
[2025-02-05T12:17:17.953+0000] {logging_mixin.py:188} INFO - 12:17:17  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T12:17:17.967+0000] {logging_mixin.py:188} INFO - 12:17:17
[2025-02-05T12:17:17.968+0000] {logging_mixin.py:188} INFO - 12:17:17  Nothing to do. Try checking your model configs and model specification args
[2025-02-05T12:17:18.753+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T12:17:18.755+0000] {local.py:494} INFO - Inlets: []
[2025-02-05T12:17:18.756+0000] {local.py:495} INFO - Outlets: []
[2025-02-05T12:17:18.757+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T12:17:18.758+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T12:17:18.768+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_sales to None, run_after=None
[2025-02-05T12:17:18.806+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T12:17:18.814+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_sales, task_id=model_sales.***_test, run_id=manual__2025-02-05T12:16:26+00:00, execution_date=20250205T121626, start_date=20250205T121715, end_date=20250205T121718
[2025-02-05T12:17:18.841+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T12:17:18.855+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T12:17:18.857+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
